task/06-conditions-n-loops-tasks.js

```diff
@@ -30,10 +29,18 @@
  *
  */
 function getFizzBuzz(num) {
-    throw new Error('Not implemented');
+  const isFizz = num % 3 === 0;
+  const isBuzz = num % 5 === 0;
+
+  return isFizz && isBuzz
```

Comment:
More than 1 ternary operator in a row is really hard to read and understand. Please rewrite this function